### ISSUE TITLE
cannot scroll with arrow keys after navigating to a room

### ISSUE THREAD

**Issue by trevorah** [gitterHQ/gitter#300]
The highlighted room in the list gets moved instead

**Comment by fperez** [comment]
Any progress on this one? I should add that I only see the problem on Chrome+Linux, where I also see that scrolling of the main window with touchpad or mouse wheel basically doesn't work at all.

On Firefox+Linux, it works reasonably well. So this may be either a chrome, or chrome+linux-specific problem.

Still, would be great if it could be solved. Right now the experience with that browser/OS combo is a bit miserable.

Otherwise thanks for the great tool!

**Comment by MadLittleMods** [comment]
This is resolved with the [new left menu release](http://blog.gitter.im/2016/05/12/a-new-look-for-gitter/)
